astro/src/content/blog/fusionauth-on-render.mdx

```diff
@@ -0,0 +1,219 @@
+
+## Setting Up PostgreSQL On Render
+
+First things first, you need to set up our PostgreSQL database.
+
+### Create A New PostgreSQL Database
+
+1. Go to your Render dashboard and create a new PostgreSQL instance.
+2. Name this instance `fusionauth-example`.
+    
+    ![PostgreSQL](/img/blogs/fusionauth-on-render/1-fusionauth-render.jpg)
+3. Choose `Ohio (US East)` as the region.
+4. Select `PostgreSQL 16`, which is great for hobby projects.
+5. Set it to the `Free` tier for now. This will work for any development work, you will want to upgrade if utilizing the database in production, mainly for the zero downtime aspects.
+    
+![PostgreSQL Instance Type](/img/blogs/fusionauth-on-render/2-fusionauth-render.jpg)
+
+Now, let's create the database. While it’s setting up, you can proceed with setting up FusionAuth.
+
+## Setting Up FusionAuth On Render
+
+### Create A New Web Service For FusionAuth
+
+Next, you need to set up FusionAuth as a web service.
+
+1. In your Render dashboard, create a new web service.
+2. Use an existing image, hosted on docker.
+3. Specify the image: `docker.io/fusionauth/fusionauth-app:latest`.
+    
+```plaintext
+docker.io/fusionauth/fusionauth-app:latest
+```
+4. Choose `Ohio (US East)` as the region again.
+5. For initial testing and development, set the service to the `Free` tier (there is a limitation with this that I show in the video at a later point).
+    
+### Configure Environment Variables
+
+FusionAuth requires several [environment variables](/docs/get-started/download-and-install/docker) to be set when using Docker.
+
+![Environment Variables](/img/blogs/fusionauth-on-render/3-fusionauth-render.jpg)
+
+Here’s a list:
+
+- `DATABASE_URL`: PostgreSQL database URL
+- `DATABASE_USERNAME`: Database username
+- `DATABASE_PASSWORD`: Database password
+- `FUSIONAUTH_APP_MEMORY`: Memory used for Java App (only required for limited workloads)
+- `FUSIONAUTH_APP_RUNTIME_MODE`: Development Mode or Production
+- `OPENSEARCH_JAVA_OPTS`: Java Options (only required for limited workloads)
+- `POSTGRES_USER`: Postgres user
+- `POSTGRES_PASSWORD`: Postgres password
+
+Pull this information from your PostgreSQL setup, and apply it to these environment variables.
+
+Example values below
+```plaintext
+DATABASE_URL=jdbc:postgresql://hostname:5432/fusionauth-example
+DATABASE_USERNAME=fusionauth_example_user
+DATABASE_PASSWORD=2Ss7JqaLDSCrjSFwqoLAWyvn2q8764654
+FUSIONAUTH_APP_MEMORY=512M
+FUSIONAUTH_APP_RUNTIME_MODE=development
+OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
+```
+
+### Deploy The Web Service
+
+Click the `Deploy Web Service` button. Wait for the deployment process to complete. You’ll see a loading screen as the service starts.
+
+![Logs showing in Render](/img/blogs/fusionauth-on-render/6-fusionauth-render.jpg)
+
+Once the logs show `HTTP server listening on port [9011]` you will be ready to access the web service.
+
+### Enter Maintenance Mode
```

Comment:
Would it be easier to use a kickstart here?

Reply:
That is not possible. This has to be a docker image.